packages/react-devtools-shared/src/backend/views/TraceUpdates/canvas.js

```diff
@@ -11,6 +11,8 @@ import type {Data} from './index';
 import type {Rect} from '../utils';
 import type {NativeType} from '../../types';
 
+import Agent from 'react-devtools-shared/src/backend/agent';
```

Comment:
nit: I'm not sure, are we able to import `type Agent` here?

Reply:
I think either works, but I agree import only type is better as that doesn't affect output of the JS bundle, and I am not actually creating Agent incident here. I'll make changes.